src/Munee/Asset/Type/Css.php

```diff
@@ -204,4 +205,42 @@ protected function fixRelativeImagePaths($content, $originalFile)
 
         return $content;
     }
+
+    private static function parseImports($content,$origFile)
+    {
+        $dir   = dirname($origFile);
+        // matches any type of import rule
+        preg_match_all('~@import\s*(?:url)?(?:\(?\'?\"?)?([^\'\"\)\(]*)(?:\'?\"?)?\)?\s?([^;]*);~im',$content,$imports,PREG_SET_ORDER);
+        if(count($imports))
```

Comment:
I do not believe you need this if statement.  The `preg_match_all` will create the `$imports` as a blank array if it doesn't match anything.  Which means this loop won't run anyway.